HomeWork1/task2.py

```diff
@@ -0,0 +1,13 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+number = int(input("Введите положительное число: "))
+sum = 0
+while number > 0:
+    sum = sum+number % 10
+    number = number // 10
```

Comment:
Можно использовать короткую запись
```suggestion
    sum += number % 10
    number //= 10
```